### DISCUSSION TITLE
"The following pull requests may not be able to be merged if this run is deleted"

### DISCUSSION THREAD

**Discussion by allenrobel** [community/community#143041] [Actions] [answered]
### Select Topic Area

Question

### Body

When deleting _some_ (not all) workflow runs, I see the warning below.  What does it mean (especially since the PR has already been merged, and the corresponding branch is deleted ).

I assume it's safe to delete the workflow run, but what can cause this?

Thanks!
Allen

![image](https://github.com/user-attachments/assets/d75f60e3-0900-4031-8202-ea145c81c7f9)

**Comment by ianshulx** [comment, marked as answer]
The warning means that deleting this workflow run will erase its history, which might be useful for tracking or auditing. But since the pull request is already merged and the branch is deleted.

Is It Safe to Delete?
Yes, it should be safe to delete the workflow run, especially if:

The pull request is merged.
The branch associated with the PR is deleted.
Deleting a run won’t affect the merged code or the repository itself. However, the only downside is losing that specific workflow's record if you need it for future reference.

**Reply by allenrobel** [reply]
Thank you Anshul!
